examples/application/attachedcluster2.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: cluster.kurator.dev/v1alpha1
+kind: AttachedCluster
+metadata:
+  name: ac2
```

Comment:
kurator create `kurator-member1` and `kurator-member2` cluster with `hack/local-dev-setup.sh`, you'd better resue these clusters.

Reply:
will check if 'kurator-member1','kurator-member1' can as example here